web/ui/src/features/widgets/Table.module.css

```diff
@@ -0,0 +1,34 @@
+.table {
```

Comment:
I hesitate to name any folder `widget` and `component` as developers usually use these name to dump all generic reusable components into, but in this repo the folders seem to only pertain to one ui element, perhaps name it /table